src/js/view/controls/components/menu/menu.js

```diff
@@ -65,27 +65,6 @@ export default class Menu extends Events {
             const key = sourceEvent && normalizeKey(sourceEvent.key);
             let focusEl;
 
-
-            switch (sourceEvent.keyCode) {
```

Comment:
Why were these moved to tizen? How are these inputs implemented for desktop now?

Reply:
The original implementation was never changed so they still exist for desktop right below. For the first app, this was a quick workaround for navigation through the settings-menu. This is just some cleanup from the first app.